.github/workflows/http-python-requests.yml

```diff
@@ -0,0 +1,77 @@
+name: Python HTTP Requests
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+    paths: ['http/python/requests/**', '.github/workflows/**']
+  pull_request:
+    paths: [http/python/request/**', '.github/workflows/**']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      relativePath: ./http/python/requests
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.4.0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2.3.2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv
+          pipenv install --dev --skip-lock
```

Comment:
assuming the --dev means it'll look for abstractions on the test feed?

Reply:
Nope `--dev` means to install developer dependencies as well e.g static type checker, linter, test runner etc.

The package index from which to install a dependency is specified in the `Pipfile` like so `kiota-abstractions = {version="*", index="testpypi"}`

Reply:
thanks for the precision, resolving.